src/libraries/restore/runtime/runtime.depproj

```diff
@@ -5,22 +5,17 @@
     <RuntimeIdentifier Condition="'$(RuntimeOS)' == 'webassembly'">$(ToolRuntimeRID)</RuntimeIdentifier>
     <NoWarn>$(NoWarn);NU1603;NU1605</NoWarn>
     <SwapNativeForIL Condition="'$(SwapNativeForIL)' == '' and ('$(ConfigurationGroup)' == 'Debug' or '$(Coverage)' == 'true')">true</SwapNativeForIL>
+    <ImportLiveBuildsTargets>true</ImportLiveBuildsTargets>
```

Comment:
Why we don't import them always?

Reply:
In the initial design I decided to make it opt-in, but I don't entirely know why and can't think of a good justification with the final design. I'll remove the opt-in and make it always-imported.